lib/defaults/defaults.go

```diff
@@ -211,10 +211,10 @@ const (
 	GravityPackagePrefix = "gravitational.io/gravity"
 
 	// TelekubeSystemLog defines the default location for the system log
-	TelekubeSystemLog = "./telekube-system.log"
+	TelekubeSystemLog = "/var/log/telekube-system.log"
```

Comment:
why this change?

Reply:
I explained it in the PR description.

> This gives us a stable well-known location where these files can be found, as opposed to currently having to search for them everywhere because the working dir may be completely different depending on how the command was run (e.g. if it's run via an agent, then the file will end up in / and so on).

Reply:
ok, makes sense